doctr/models/resnet.py

```diff
@@ -0,0 +1,138 @@
+# Copyright (C) 2021, Mindee.
+
+# This program is licensed under the Apache License version 2.
+# See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
+
+import tensorflow as tf
+from tensorflow.keras import layers
+from tensorflow.keras.models import Sequential
+from typing import Tuple
+
+__all__ = ['Resnet31']
+
+
+class ResnetBlock(layers.Layer):
+    """Implements a resnet31 block with shortcut
+
+    Args:
+        conv_shortcut: Use of shortcut
+        output_channels: number of channels to use in Conv2D
+        kernel_size: size of square kernels
+
+    """
+    def __init__(
+        self,
+        output_channels: int,
+        conv_shortcut: bool,
+        **kwargs
+    ) -> None:
+
+        super().__init__(**kwargs)
+        if conv_shortcut:
+            self.shortcut = Sequential(
+                [
+                    layers.Conv2D(
+                        filters=output_channels,
+                        padding='same',
+                        kernel_size=1,
+                        use_bias=False,
+                        kernel_initializer='he_normal'
+                    ),
+                    layers.BatchNormalization()
+                ]
+            )
+        else:
+            self.shortcut = layers.Lambda(lambda x: x)
+        self.conv_block = Sequential(
+            self.conv_resnetblock(output_channels, 3)
+        )
+        self.act = layers.Activation('relu')
+
+    @staticmethod
+    def conv_resnetblock(output_channels, kernel_size):
+        return [
+            layers.Conv2D(output_channels, kernel_size, padding='same', use_bias=False, kernel_initializer='he_normal'),
+            layers.BatchNormalization(),
+            layers.Activation('relu'),
+            layers.Conv2D(output_channels, kernel_size, padding='same', use_bias=False, kernel_initializer='he_normal'),
+            layers.BatchNormalization(),
```

Comment:
Could you use `conv_sequence` instead please? (kernel initializer, bias, and padding are already set correctly by default)